src/components/itemCard.js

```diff
@@ -25,9 +30,59 @@ export default function ItemCard({
     duedate,
     id: item.id,
   })
+  const [timeLeft, setTimeLeft] = useState({})
+
+  function calculateTimeLeft() {
+    const deadline = new Date(duedate)
+    deadline.setDate(deadline.getDate() + 1)
+    const now = new Date()
+    const difference = deadline - now
+
+    setTimeLeft({
+      days: Math.floor(difference / (1000 * 60 * 60 * 24)),
+      hours: Math.floor((difference / (1000 * 60 * 60)) % 24),
+    })
+  }
+  useEffect(() => {
+    setTimeout(() => {
+      calculateTimeLeft()
+    }, 1000)
+  })
```

Comment:
You forgot to provide a cleanup function, this produces a warning in the console, when the item is dismissed. Try this
```
useEffect(() => {
    const timer = setTimeout(() => {
      calculateTimeLeft()
    }, 1000)
    return () => clearTimeout(timer)
  })
```